database-mysql/index.js

```diff
@@ -32,6 +32,23 @@ connection.sync()
     console.log(err, '!!!!!!!!!!!!!');
   });
 
+  Users.create({
+    name: 'Kalkidan',
+    userName: 'KalkidanMulatu',
+    profilePic: 'https://scontent-dfw5-1.xx.fbcdn.net/v/t1.0-0/p110x80/48359292_220566528875309_249279966284349440_n.jpg?_nc_cat=104&_nc_ht=scontent-dfw5-1.xx&oh=c87c3905e336d7235594112d4b2d3db0&oe=5D91B150' 
+  })
+  Users.create({
+    name: 'Kelly',
+    userName: 'KellyT',
+    profilePic: 'https://scontent-dfw5-1.xx.fbcdn.net/v/t1.0-0/p110x80/48359292_220566528875309_249279966284349440_n.jpg?_nc_cat=104&_nc_ht=scontent-dfw5-1.xx&oh=c87c3905e336d7235594112d4b2d3db0&oe=5D91B150' 
```

Comment:
you really gonna put your pic on this site?

Reply:
NO. I thought this is a dummy data for us to use for now